act-rules-format.bs

```diff
@@ -140,43 +140,37 @@ Component properties starting with `aria-` MUST exist in the list of WAI-ARIA at
 
-Selector syntax depends on the test subject type. When a formal syntax can be used, that (part of the) selector should use that syntax. When a formal selector syntax can not be used, that (part of the) selector MAY be described unambiguously in plain language.
+Applicability MUST be described objectively, unambiguously and in plain language. When a formal syntax (such as a CSS selector or XPATH) can be used, that (part of the) applicability MAY use that syntax in addition to the plain language description. While testing, if nothing on in the test subject matches the applicability of the rule, the result is `inapplicable`.
 
-This can be used together, such as in the following example which uses CSS selector syntax to locate elements, combined with a plain language description to further filter the nodes.
+An objective description is one that can be resolved without uncertainty in a given technology. Examples of objective properties in HTML are element names, their computed role, the spacing between two elements, etc. Subjective properties on the other hand, are concepts like decorative, navigation mechanism and pre-recorded. Even concepts like headings and images can be misunderstood. Is this talking about the tag name, the accessibility role, it's purpose in the web page? The later of which is almost impossible to define with objectively. When used in applicability, these concepts MUST have an objective definition. This definition can be part of a larger glossary shared between rules.
 
 <div class=example>
-A rule for HTML `img` elements has the following selector:
+A rule for labels of HTML `input` elements may have the following applicability:
 
-1. Select elements matching the following CSS Selector: `img:not([alt=""])`
-2. Exclude any elements that have `visibility:hidden` or `display: none`
+Each *available* `input` element, except elements with `type="hidden"`.
 
+**available**: The isn't hidden with `display:none`, `visibility:hidden` or though `aria-hidden="true"`.
 </div>
 
 
-Test Execution Steps {#test-proc-steps}
------------------------------
-
-Each rule contains one or more test execution steps written in plain language. Each test execution step MUST provide the following:
+Expectations
+------------
 
-* Test mode identifying whether test execution step is automated, semi-automated or requires manual testing.
-* Test execution which clearly outlines the actions necessary to reach the result.
-* Test result indicating whether test execution step passed or failed.
-  * Description of the error causing any failing result.
+Each rule MUST contain one or more expectations. An expectation is a statement that must be true about each *test target* (see Applicability). Each expectation MUST be unambiguous and be written in plain language. Unlike in applicability, a certain level of subjectively is allowed in expectations. Meaning that the expectation has only one possible meaning, but that meaning isn't fully quantifiable.
```

Comment:
"subjectively" --> "subjectivity"

Reply:
We had a discussion to add that each Expectation is a Conformance statement and should be testable on its own. This will clarify what we mean by Expectations.